src/docs/asciidoc/architecture/02_business_concepts/03_business_services/01_user.adoc

```diff
@@ -29,15 +29,14 @@ Description:: Create User
 Pre requisite:: User must be defined in provided authentication service
 
 .Nominal Scenario
- . User succeed in connecting for the first time (See To Authenticate)
- . A user is registred in the system
+ . Automation creation user when an user succeed in authenticating and connecting for the first time (See <<To Authenticate>>)
 
 .Alternative scenario
- . Administrator creates a user directly
+ . Administrator creates an user directly
 
 .Error scenario
- * If user is not registred in authentication provider when an Administrator creates it, raise an error
- * If user already exist when an Administrator creates it, raise an error
+ * If an user is not registered in authentication provider when an Administrator creates it, raise an error
```

Comment:
English language is tricky about the "a/an" usage and it seems that for nouns beginning with a "u" which sounds like "y" in you or a "o" which sounds like "w" in "won", a "a" should be used (cf [grammaticality - Is it "a user" or "an user" - English Language & Usage Stack Exchange](https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user))